content/first-order-logic/completeness/completeness-thm.tex

```diff
@@ -42,15 +42,15 @@
 \Gamma$, $\iftag{FOL}{\Sat{M(\Gamma^*)}}{\pSat{v(\Gamma^*)}}{!A}$, so
 $\Gamma$ is satisfiable.\iftag{FOL}{
   If $\Gamma$ does contain~$\eq$,
-  then by \olref[ide]{lem:truth}, $\Sat{M/_\approx}{!A}$ iff $!A \in
+  then by \olref[ide]{lem:truth}, $\Sat{M/\mathord\approx}{!A}$ iff $!A \in
```

Comment:
Should actually be `\equivclass{M}{\approx}` -- that makes the typesetting of equivalence classes uniform.